src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -335,12 +339,10 @@ public static PathKind GetPathKind(string path)
             return PathKind.Relative;
         }
 
-#nullable enable
-
         /// <summary>
         /// True if the path is an absolute path (rooted to drive or network share)
         /// </summary>
-        public static bool IsAbsolute([NotNullWhen(true)]string? path)
+        public static bool IsAbsolute([NotNullWhen(returnValue: true)] string? path)
```

Comment:
>returnValue: [](start = 51, length = 12)

Why add this? Seems unnecessary

Reply:
We use it in most other places. I don't much care though

---
In reply to: [377805204](https://github.com/dotnet/roslyn/pull/41063#discussion_r377805204) [](ancestors = 377805204)